ui/src/containers/DashboardLayout/index.tsx

```diff
@@ -14,7 +17,50 @@ interface LayoutProps {
   Header: React.FC;
 }
 
+const adminSection = (
+  <>
+    <div className={c('header-row')}>Admin</div>
+    <Link href="/organizations">
+      <div className={c('row')}>
+        <MdBusiness />
+        <span className={c('row-text')}>Organizations</span>
+      </div>
+    </Link>
+
+    <Link href="/employees">
+      <div className={c('row')}>
+        <FaUsers />
+        <span className={c('row-text')}>Employees</span>
+      </div>
+    </Link>
+  </>
+);
+
+const project = (item: any) => {
```

Comment:
Please refrain from using `any` type unless absolutely necessary. Here the type should be `Project` I think.